exonum-java-binding-core/rust/integration_tests/tests/service_proxy.rs

```diff
@@ -1,32 +1,47 @@
+extern crate exonum_testkit;
 extern crate integration_tests;
 extern crate java_bindings;
 #[macro_use]
 extern crate lazy_static;
+#[macro_use]
+extern crate serde_derive;
+
+use std::{
+    panic::{catch_unwind, AssertUnwindSafe},
+    sync::Arc,
+};
 
-use integration_tests::mock::service::ServiceMockBuilder;
-use integration_tests::mock::transaction::{create_mock_transaction, INFO_VALUE};
-use integration_tests::test_service::{
-    create_test_map, create_test_service, INITIAL_ENTRY_KEY, INITIAL_ENTRY_VALUE,
+use exonum_testkit::TestKitBuilder;
+
+use integration_tests::{
+    mock::{
+        service::ServiceMockBuilder,
+        transaction::{create_mock_transaction, INFO_VALUE},
+    },
+    test_service::{create_test_map, create_test_service, INITIAL_ENTRY_KEY, INITIAL_ENTRY_VALUE},
+    vm::create_vm_for_tests_with_fake_classes,
+};
+use java_bindings::{
+    exonum::{
+        blockchain::Service,
+        crypto::hash,
+        encoding::Error as MessageError,
+        messages::RawTransaction,
+        storage::{Database, MemoryDB},
+    },
+    jni::{objects::JObject, JavaVM},
+    serde_json::{from_str, Value},
```

Comment:
Perhaps a personal taste, but I don't like imports from `serde_json`. That's because when you find `let smth = from_str(smth_else)` - it's hard to understand why you can actually convert `smth_else` to `smth`

```suggestion
    serde_json,
```

Reply:
Well, mee too ) But I accepted this for tests 😁. Ok, I agree - rewriting it.